agenda/presentation/src/main/java/com/nibalk/tasky/agenda/presentation/components/AgendaCard.kt

```diff
@@ -82,13 +80,13 @@ fun AgendaCard(
                 onItemClick()
             },
         elevation = CardDefaults.cardElevation(defaultElevation = 2.dp),
-        colors = CardDefaults.cardColors(containerColor = backgroundColor)
+        colors = CardDefaults.cardColors(containerColor = cardColor.first)
     ) {
         // Title Row
         AgendaCardHeader(
             title = item.title,
             isDone = if (item is AgendaItem.Task) item.isDone else false,
-            contentColor = contentColor,
+            contentColor = cardColor.second,
```

Comment:
And replacing this here accordingly. Otherwise, reading `cardColor.second` doesn't tell you what kind of color that is without having to check the when expression